pages/api/module/post.ts

```diff
@@ -0,0 +1,24 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { Module } from "../models/module";
+
+const post = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    let postModule;
+    if (!req.body.title)
+        return res.status(400).send("Please provide a title for your module.");
+    else {
```

Comment:
You can get rid of the `else` block here since you're returning in the if block, and so you can assign `postModule` directly